documents/transfer.md

```diff
@@ -0,0 +1,572 @@
+## Transfers
+
+### Create transfers from payment
+
+```php
+$api->payment->fetch($paymentId)->transfer(array('transfers' => array('account'=> $accountId, 'amount'=> '1000', 'currency'=>'INR', 'notes'=> array('name'=>'Gaurav Kumar', 'roll_no'=>'IEC2011025'), 'linked_account_notes'=>array('branch'), 'on_hold'=>'1', 'on_hold_until'=>'1671222870')));
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| paymentId*   | string      | The id of the payment to be fetched  |
+| transfers   | array     | All parameters listed [here](https://razorpay.com/docs/api/route/#create-transfers-from-payments) are supported |
+
+**Response:**
+```json
+{
+  "entity": "collection",
+  "count": 1,
+  "items": [
+    {
+      "id": "trf_E9uhYLFLLZ2pks",
+      "entity": "transfer",
+      "source": "pay_E8JR8E0XyjUSZd",
+      "recipient": "acc_CPRsN1LkFccllA",
+      "amount": 100,
+      "currency": "INR",
+      "amount_reversed": 0,
+      "notes": {
+        "name": "Gaurav Kumar",
+        "roll_no": "IEC2011025"
+      },
+      "on_hold": true,
+      "on_hold_until": 1671222870,
+      "recipient_settlement_id": null,
+      "created_at": 1580218356,
+      "linked_account_notes": [
+        "roll_no"
+      ],
+      "processed_at": 1580218357
+    }
+  ]
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Create transfers from order
+
+```php
+$api->payment->fetch($paymentId)->transfer('amount'=>'1000', 'currency'=>'INR', array('transfers' => array('account'=> $accountId, 'amount'=> '100', 'currency'=>'INR', 'notes'=> array('branch'=>'Acme Corp Bangalore North', 'name'=>'Gaurav Kumar'), 'linked_account_notes'=>array('branch'), 'on_hold'=>'1', 'on_hold_until'=>'1671222870')));
```

Comment:
@HancilSequeira this should be order create and then transfer, this looks same to create transfer from payments above method